benchmarks/README.md

```diff
@@ -0,0 +1,64 @@
+# sgkit Benchmarks
+
+Benchmarking sgkit with Airspeed Velocity.
+
+## Usage
+
+Airspeed Velocity manages building the environment via conda itself. The recipe for the same in
+defined in the `asv.conf.json` configuration file.
+
+* To run the benchmark suite:
+
+```bash
+asv run --config benchmarks/asv.conf.json
```

Comment:
Do I have to do anything extra to install asv?

Reply:
Nothing extra apart from installing `requirements-dev.txt`